core/src/main/java/hudson/model/Run.java

```diff
@@ -1852,7 +1852,7 @@ protected final void execute(@Nonnull RunExecution job) {
 
                     setResult(job.run(listener));
 
-                    LOGGER.log(INFO, "{0} main build action completed: {1}", new Object[] {this, result});
+                    LOGGER.log(FINEST, "{0} main build action completed: {1}", new Object[] {this, result});
```

Comment:
Probably a good idea: It's time we started removing this from the general system log, since it doesn't really matter to admins.

Reply:
I agree with it as well

Reply:
I think `FINE` would have been appropriate for something which is relevant and not emitted all that frequently. `FINEST` should really be reserved for only the lowest-level tracing of the behavior of individual methods.

Reply:
This broke a lot of our monitoring as we have elastalert checking for "main build action completed: SUCCESS" for our cronjobs and if we don't have any in 15 minutes it triggers an error. The log was previously being collected from stdout/stderr on the container running jenkins but now there doesn't seem to be a way to get it to output the value